### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change

I've found some additional updates in the README, etc. that should be fixed, will continue to review our documentation to see where we can be more explicit (I suspect feedback post-open source launch will help here as well).

### Alternate Designs

None, this is just docs updates not code.

### Benefits

Hopefully less support requests and easier setup for new users/contributors.

### Possible Drawbacks

I could be using more confusing language, but hopefully a review will squash any of those issues/concerns.

### Verification Process

I checked the edits in the GitHub Preview UI.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/classifai-for-wordpress/blob/develop/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues

n/a

### CODE REVIEW

**Comment by jeffpaul** [comment]
Looks like we could move Code of Conduct and Contributing markdown files to the .github folder as well, though I like that they're more readily viewable in the root so keeping them there for now.

**Comment by jeffpaul** [comment]
I've noticed several references to "Classifai" in the code (mostly comments), considering updating those all to "ClassiAI" to match our capitalization (capital_AI_dangit).

**Comment by jeffpaul** [comment]
With us renaming the repo to `classifai`, I'll add to this PR with docs cleanup to cover that change.

File: classifai.php [hunk 1]
```diff
@@ -1,13 +1,15 @@
 <?php
 /**
  * Plugin Name:     ClassifAI
- * Description:     AI-powered classification and machine learning for WordPress content
+ * Plugin URI:      https://github.com/10up/classifai-for-wordpress
+ * Description:     Enhance your WordPress content with Artificial Intelligence and Machine Learning services.
+ * Version:         1.2.0
```
Comment (helen) [inline classifai.php]:
```suggestion
 * Version:         1.2.1
```

**Pull request merged**